resources/views/layouts/default/flights/show.blade.php

```diff
@@ -6,7 +6,14 @@
     <div class="col-8">
       <div class="row">
         <div class="col-12">
-          <h2>{{ $flight->ident }}</h2>
+          <h2>
+            {{ $flight->ident }}
+            @if ($acars_plugin && $bid)
+              <a href="vmsacars:bid/{{$bid->id}}" class="btn btn-info btn-sm float-right">Load in vmsACARS</a>
+            @elseif ($acars_plugin)
+              <a href="vmsacars:flight/{{$flight->id}}" class="btn btn-info btn-sm float-right">Load in vmsACARS</a>
+            @endif
+          </h2>
         </div>
```

Comment:
Try this

```
<div class="col">
          <h2>{{ $flight->ident }}</h2>
        </div>
<div class="col">
@if ($acars_plugin && $bid)
              <a href="vmsacars:bid/{{$bid->id}}" class="btn btn-info btn-sm float-right">Load in vmsACARS</a>
            @elseif ($acars_plugin)
              <a href="vmsacars:flight/{{$flight->id}}" class="btn btn-info btn-sm float-right">Load in vmsACARS</a>
            @endif
          </div>
```

Reply:
Yeah, if you wish we can divide that row to two columns and separate them. Then we may remove the float-right too and make that second column aligned right.

Putting buttons and badges inside the <h> tags makes them auto sized according to it though, this is why I use them mostly this way but this is default theme, so your choices have priority.

Shall I split them up now ?